srcjs/bindings/treeviewInputBinding.js

```diff
@@ -73,7 +75,8 @@ $.extend(f7TreeviewBinding, {
     }
```

Comment:
@DivadNojnarg not sure what is the most logical behaviour here:

1. if you select the group title, you only get the parent returned in the input (current)
2. if you select the group title, all children get returned in the input (parent excluded)

For the checkbox variant it's currently 2, but the checkbox behaviour is different because it automatically checks all children once the parent is checked (which makes sense). As per Framework 7 docs, the selectable only selects one item, which makes me lean towards 1 for the selectable. Examples for all different variations are in `/inst`.

Reply:
For the checkbox this is fine, I would expect to get all children if I select the parent. In the default case (without checkbox), I would select the parent only: if it is a file path for instance that would make sense to only select the element clicked.